### PR TITLE
Fallback on create & update actions

### PR DESCRIPTION
**Pull request by panyamin**
Fallback for create and update to new and edit actions, because they are probably duplicates.

### CODE REVIEW

**Comment by panyamin** [comment]
Should I follow HoundCI's style convention?

File: app/helpers/title/title_helper.rb [hunk 1]
```diff
@@ -8,7 +8,15 @@ def title(additional_context = {})
...
-        @action_name = action_name
+        @action_name = case action_name
+          when 'create'
+            'new'
+          when 'update'
+            'edit'
+          else
+            action_name
+          end
+
```
Comment (calebthompson) [inline app/helpers/title/title_helper.rb]: I don't think this is an issue. I like the extra space after the case.

**Comment by calebthompson** [comment]
If you wouldn't mind. Another alternative would be:

```ruby
action_names = Hash.new { |hash, key| hash[key] = key }.merge("create" => "new", "update" => "edit"
```

The problem I see with this change is that if someone has actually specified a title for update or create views, they'll want that used instead of new/edit defaults. I don't think this would manage that case?

**Comment by panyamin** [comment]
You're right.
I thought that in a normal way, handle `create` and `update` actions - which render the same views as of `new` and `edit` actions -, the title stay the same.
If you want I can try to code something accordingly.

**Comment by calebthompson** [comment]
This is certainly an improvement. Let me ping a few people using Title and see if this will break anything for them. I'm definitely thinking that this is an improvement and my case is a minority, if anyone.

**Comment by calebthompson** [comment]
We'll want a README update to mention either the mapping (as you've written it) or a fallback (if it changes), as well as a spec to verify the behavior.

**Comment by panyamin** [comment]
Ok, I'll update the commit asap.
I have an *ugly* fix in mind in order to have a specific title for these actions: use the `default:` option in I18n.translate (see http://guides.rubyonrails.org/i18n.html#defaults). What do you think?

**Comment by SirRawlins** [comment]
This looks like a really tidy solution guys - nice work.

**Pull request merged**